### DISCUSSION TITLE
'node-click': ((...args: unknown[]) => void) | undefined - Is this a bug or did I use it incorrectly?

### DISCUSSION THREAD

**Discussion by xxxace** [bcakmakoglu/vue-flow#1256] [Q&A] [answered]
![image](https://github.com/bcakmakoglu/vue-flow/assets/60084325/52ed1ce7-fdb8-4c61-a583-5cd87c8a6649)

Type '(e: NodeMouseEvent) => void' is not assignable to type '(...args: unknown[]) => void'.
  Types of parameters 'e' and 'args' are incompatible.ts(2322)

Currently, I am using this way to fix it.
![image](https://github.com/bcakmakoglu/vue-flow/assets/60084325/bff9cc9b-ec3c-4733-8e7b-9d25a655a905)

**Comment by bcakmakoglu** [comment]
Not sure *where* you get that error from - could you elaborate more on that?

**Reply by xxxace** [reply, marked as answer]
Thanks for the reply.

![image](https://github.com/bcakmakoglu/vue-flow/assets/60084325/1f4e432f-4524-4c64-928e-afdddee101be)

now i use useVueFlow  to avoid this.

`const { onNodeDoubleClick } = useVueFlow({})`
